replibyte/src/destination/postgres_docker.rs

```diff
@@ -0,0 +1,75 @@
+use crate::connector::Connector;
+use crate::destination::Destination;
+use crate::types::Bytes;
+use crate::utils::binary_exists;
+use std::io::{Error, ErrorKind, Write};
+use std::process::{Command, Stdio};
+
+pub struct PostgresDocker {}
+
+impl Default for PostgresDocker {
+    fn default() -> Self {
+        PostgresDocker {}
+    }
+}
+
+impl Connector for PostgresDocker {
+    fn init(&mut self) -> Result<(), Error> {
+        let _ = binary_exists("docker")?;
+
+        // Start the container and configure the network and credentials to be able to connect locally.
+        // TODO: use free port, handle postgres version and container name.
```

Comment:
maybe the port can be also configurable by the user

Reply:
👍 I've added an optional `--port` flag